### ISSUE TITLE
Dependency vulnerability using `cargo audit`

### ISSUE THREAD

**Issue by LukeMathWalker** [sru-systems/rust-argon2#16]
Using `cargo audit` on a project, I get
```
error: Vulnerable crates found!

ID:	 RUSTSEC-2019-0011
Crate:	 memoffset
Version: 0.2.1
Date:	 2019-07-16
URL:	 https://github.com/Gilnaa/memoffset/issues/9#issuecomment-505461490
Title:	 Flaw in offset_of and span_of causes SIGILL, drops uninitialized memory of arbitrary type on panic in client code
Solution: upgrade to: >= 0.5.0

error: 1 vulnerability found!
```

It is being pulled in from `crossbeam-epoch` v0.5 - could you update your dependencies to solve it?

**Comment by repi** [comment]
Ran into this as well, thanks for fixing it! Switched to use your fork for now

**Comment by LukeMathWalker** [comment]
Do you mean @kpcyrd's fork, @repi?

**Comment by repi** [comment]
Yes!

**Comment by kpcyrd** [comment]
@repi please be very careful with that, I tend to delete these branches after they've been merged. I'd rather see this addressed in a new version published to crates.io.

**Comment by repi** [comment]
> @repi please be very careful with that, I tend to delete these branches after they've been merged. I'd rather see this addressed in a new version published to crates.io.

Thanks for the heads up! And yes would prefer a new published release also, but this was a blocker issue for us as our CI fails if `cargo audit` fails. We could always fork it ourselves though

**Comment by mrijkeboer** [comment]
@LukeMathWalker 

Thanks for creating this issue. This issue has been resolved with merging pull request #19.
